packages/components/src/internal/components/editable/models.ts

```diff
@@ -427,11 +444,20 @@ export class EditorModel
         return {
             uniqueKeyViolations,
             missingRequired,
+            cellMessages,
         };
     }
 
-    getValidationErrors(queryModel: QueryModel, uniqueFieldKey?: string, insertColumns?: QueryColumn[]): string[] {
-        const { uniqueKeyViolations, missingRequired } = this.validateData(queryModel, uniqueFieldKey, insertColumns);
+    getValidationErrors(
```

Comment:
The only way for the update case to get a missing value cell warning is if you explicitly interacted with (3, int) cell. It's more likely that a user will forget to fill in required values during add, and much less likely that they would intentionally blank out required during update. I think I'll leave this as is since client side required value check was never wired up for the update case.

Reply:
Good catch. I've wired up required value check for the update case via cell interation: delete, cut, clear, etc.

Reply:
This looks much better. thank you for wiring that up. 

However, when I select some cells > Edit in Grid > remove some required values > click Finish, I am getting JS errors for `value.toLowerCase is not a function`

<img width="1194" alt="Screenshot 2024-07-11 at 7 10 16 AM" src="https://github.com/LabKey/labkey-ui-components/assets/6411206/1d775bbb-0cf3-4c8e-94b6-b91222211ae1">

Reply:
looks good